index.js

```diff
@@ -168,7 +194,71 @@ app.all('*', (req, res) => {
     }
   }
 
-  const errorMessage = `Request ${req.url} didn't match any registered route.`;
+  const obfuscatedBody = JSON.parse(JSON.stringify(req.body));
+  recordingsContext.deleteBodyAttributes.forEach((attr) => {
+    try {
+      console.log('Deleting attribute', attr);
+      eval(`delete obfuscatedBody.${attr}`);
+    } catch {
+      console.log('No attribute to delete: ', attr);
+    }
+  });
+  const dataToHash = {
```

Comment:
move to own function `getHashForRequest`?

Reply:
👍